projects/components/src/time-range/time-range.component.ts

```diff
@@ -67,6 +72,12 @@ import { PopoverRef } from '../popover/popover-ref';
   `
 })
 export class TimeRangeComponent {
+  @Input()
+  public dropdownLocation: PopoverRelativePositionLocation[] = [
+    PopoverRelativePositionLocation.BelowLeftAligned,
+    PopoverRelativePositionLocation.BelowRightAligned
```

Comment:
why have we given two locations as default value??

Reply:
Initial thought was that this case wouldn't be hit with the time range and would be misleading to include it there, but that's more the case for the page time range since it's always on headers, my bad, adding the above options.

Reply:
fixed